vue.json

```diff
@@ -0,0 +1,68 @@
+{
+  "extends": ["plugin:vue/vue3-essential"],
+  "parserOptions": {
+    "extraFileExtensions": [".vue"]
+  },
+  "plugins": ["vue"],
+  "rules": {
+    "vue/attributes-order": "error",
+    "vue/component-tags-order": "error",
+    "vue/no-lone-template": "error",
+    "vue/no-multiple-slot-args": "error",
+    "vue/no-v-html": "error",
+    "vue/order-in-components": "error",
+    "vue/this-in-template": "error",
+    "vue/attribute-hyphenation": "error",
+    "vue/component-definition-name-casing": "error",
+    "vue/first-attribute-linebreak": "error",
+    "vue/html-closing-bracket-spacing": "error",
+    "vue/html-end-tags": "error",
+    "vue/multiline-html-element-content-newline": "error",
+    "vue/mustache-interpolation-spacing": "error",
+    "vue/no-multi-spaces": "error",
+    "vue/no-spaces-around-equal-signs-in-attribute": "error",
```

Comment:
These are handled by Prettier. I can see them here: https://github.com/prettier/eslint-config-prettier/blob/main/index.js. I think we should remove them from our config.